### ISSUE TITLE
"Import" Keyword missing?

### ISSUE THREAD

**Issue by mistermboy** [shexSpec/grammar#13]
The [lastest grammar](https://github.com/shexSpec/grammar/blob/master/bnf) doesn't have support for the "Import" keyword. Is it right?

**Comment by labra** [comment]
I think the problem is that the latest grammar is in a branch. Maybe is it at branch [2.1](https://github.com/shexSpec/grammar/tree/shex2.1). 

At least, it is in this [ShExDoc.g4 file](https://github.com/shexSpec/grammar/blob/shex2.1/ShExDoc.g4)

Should we merge branch 2.1 to master?

**Comment by ericprud** [comment]
yeah, it's time to tackle this minor mess. @hsolbrig , i think you were committing to 2.1 most recently.
